### PR TITLE
Add info about collection documentation appearance

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
As the question about "how soon will our collection's doc appear on docs.ansible.com is frequently asked, this PR aims to clarify this.

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @felixfontein

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ New collections can be included in every minor release of the `ansible` package.
 
 For new major releases (X.0.0), new collections have to be added before the feature freeze. If they miss feature freeze, they have to wait for the next minor release (X.1.0). Deadlines for the `ansible` package can be found in the [ansible roadmaps](https://docs.ansible.com/ansible/devel/roadmap/ansible_roadmap_index.html).
 
+[The Ansible docsite](https://docs.ansible.com/ansible/devel/collections/community/index.html), the `devel` version, will refer to a collection's documentation within a day after the collection inclusion is done. The ``latest`` version will show the references after the following Ansible release.
```
Comment (felixfontein) [inline README.md:13]:
```suggestion
[The Ansible docsite](https://docs.ansible.com/ansible/devel/collections/community/index.html), the `devel` version, will refer to a collection's documentation within a day after the collection inclusion is done. The `latest` version will show the references after the following Ansible release.
```

Comment (Andersson007) [reply README.md:13]: ah, i always mix it up with rst.. thanks!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@Andersson007 thanks for improving the docs!

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!

**Pull request merged**